src/util/Constants.js

```diff
@@ -113,14 +113,8 @@ function makeImageUrl(root, { format = 'webp', size } = {}) {
   return `${root}.${format}${size ? `?size=${size}` : ''}`;
 }
 
-function makeStickerUrl(root, { format = 'png', size, passthrough = true } = {}) {
-  const params = new URLSearchParams();
-  if (size) {
-    if (!AllowedImageSizes.includes(size)) throw new RangeError('IMAGE_SIZE', size);
-    params.append('size', size);
-  }
-  params.append('passthrough', passthrough);
-  return `${root}.${format}?${params}`;
+function makeStickerUrl(root, { format = 'png' } = {}) {
+  return `${root}.${format}`;
```

Comment:
this function doesn't seem to be needed? just put the template string directly in the code

Reply:
I am agree

Reply:
this comment was marked as resolved but the function is still there?

Reply:
Sry I have forgotten to push it.